atlasdb-commons/src/test/java/com/palantir/common/compression/NotCompressedStreamTests.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.common.compression;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.NoSuchElementException;
+
+import org.junit.Test;
+
+public class NotCompressedStreamTests {
+
+    @Test(expected = NoSuchElementException.class)
```

Comment:
please don't ever use `@Test(expected` - it's super error prone (you don't get to specify _where_ in the test the exception should come out). Instead, please use `assertThatExceptionOfType(NoSucheElementException.class).isThrownBy(() -> decompressingStream.read())`